### DISCUSSION TITLE
Unable to run some phenopowerlaw simulation with multi-phase material

### DISCUSSION THREAD

**Discussion by TranMHoang** [damask-multiphysics/DAMASK#7] [Q&A] [answered]
Hi,

I am trying to run some CP simulation with phenopowerlaw using parameters given in Referenced Examples. My material is DP Ferrite and BCC Martensite. When using grid solver version 3.0.0-a8 and 3.0.0-b, the simulation returns:

`Error termination. Backtrace:
At line 175 of file /home/conda/feedstock_root/build_artifacts/damask-pkgs_1706475579011/work/src/phase_mechanical_plastic_phenopowerlaw.f90
Fortran runtime error: Attempting to allocate already allocated variable 'a_ns'`

This occurs with both Docker container and Conda Forge installation. Is it expected behavior, a solver bug, or CP parameter issue?

**Comment by sharanroongta** [comment, marked as answer]
Hi, You are right. There is a bug which is fixed in the current development version. 

If you are using the code from source (does not seem like that), I can tell you the fix and you can recompile. 

Else, you have to wait for the new release. Sorry for that.

Regards,
Sharan

**Reply by TranMHoang** [reply]
Cheers, it's nice to know that the bug is being worked on!

**Reply by GniwdulE** [reply]
Dear all, I'm working with the Phenopowelaw simulation, also with two phases and I obtained the "same" message:
At line 175 of file /home/conda/feedstock_root/build_artifacts/damask-pkgs_1706475579011/work/src/phase_mechanical_plastic_phenopowerlaw.f90

Fortran runtime error: Attempting to allocate already allocated variable 'a_ns'

Error termination. Backtrace:
#0  0x6094acc545be in ???
#1  0x6094acbe7371 in ???
#2  0x6094acbde8cc in ???
#3  0x6094acbc6238 in ???
#4  0x6094acbb8933 in ???
#5  0x6094acc6c776 in ???
#6  0x6094acb6b060 in ???
#7  0x7dc25b829d8f in __libc_start_call_main
	at ../sysdeps/nptl/libc_start_call_main.h:58
#8  0x7dc25b829e3f in __libc_start_main_impl
	at ../csu/libc-start.c:392
#9  0x6094acb6b090 in ???

Maybe you can help me. Thanks

**Reply by TranMHoang** [reply]
@GniwdulE As @sharanroongta said, it is a known bug that will be fixed in the next version. Hopefully he can give you more detailed instruction if you need to run your sims urgently.

**Reply by sharanroongta** [reply]
Hello, the new version should be hopefully released soon.
@GniwdulE If you are using the code from source, I can tell you the fix and you can recompile. Else you would have to wait for the new release.
Thanks
